cell_classification/segmentation_data_prep.py

```diff
@@ -142,7 +153,16 @@ def get_marker_activity_mask(self, instance_mask, cell_types, marker_activity):
             np.array:
                 The marker activity mask
         """
-        return np.zeros([500, 500])
+        out_list = []
+        if isinstance(markers, str):
+            markers = [markers]
+        for marker in markers:
+            out_mask = np.zeros_like(instance_mask)
+            for label, activity in enumerate(marker_activity[marker], 1):
```

Comment:
I think we can combine these two functions together, and have them handle only a single marker at a time. 

Then the input will be the conversion matrix, cell types, instance mask, and a single marker, and the output will be the out_mask for that single marker. 

This removes the need for a lot of the intermediate data structures. We can then either stack or concatenate the resulting the masks depending on how many channels we want the network to see at a time

Reply:
I merged `get_cell_types` into `get_marker_activity` so that this function handles all the tabular data lookups and returns the marker activity for a given fov and a given marker. Thus `get_marker_activity_mask` will just take instance and binary masks and the marker activity.